Libraries/LibWeb/HTML/Parser/HTMLParser.cpp

```diff
@@ -310,6 +310,9 @@ void HTMLParser::the_end(GC::Ref<DOM::Document> document, GC::Ptr<HTMLParser> pa
         (void)document->scripts_to_execute_when_parsing_has_finished().take_first();
```

Comment:
Commit message: the sentence "otherwise nowhere may end up scrolling to [..]" is a bit janky :sweat_smile:

Reply:
Thanks, will just adjust to to "nowhere", without the "may". I was thinking about the spec when I wrote that, I believe "try to scroll to the fragment" could end up scrolling to the fragment depending on the document that it is loading, but we haven't implemented that to spec yet. It's a bit of a weird spec function to be honest, I'm unclear about in which case it ends up becoming useful 😅